manager.py

```diff
@@ -118,7 +113,29 @@ def __init__(self,
             verbosity=self.v,
             logfile=self.logfile)
         # DEFAULT_UDP_PORT and DEFAULT_TCP_PORT are assigned in sender
+        self.data_handler = Data_Handler(
+            manager=self,
+            verbosity=self.v,
+            logfile=self.logfile)
 
+        self.flush_send_to_server()
+        
+    def flush_send_to_server(self, path=DEFAULT_DATA_BACKLOG_FILE):
```

Comment:
This should be a method of `DataHandler` since it is related to handling the data.